ext_localconf.php

```diff
@@ -33,6 +33,8 @@
     $GLOBALS['TYPO3_CONF_VARS']['SYS']['Objects'][\TYPO3\CMS\Backend\Form\Element\SelectSingleElement::class] = [
         'className' => \GeorgRinger\BackendDebug\Xclass\SelectSingleElement::class,
     ];
+
+    $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['t3lib/class.t3lib_pagerenderer.php']['render-preProcess'][] = 'GeorgRinger\BackendDebug\Utility\PageRenderer->setup';
```

Comment:
use ::class for the class and please don't call it PageRenderer but place it to BackendDebug\Hooks\PageRendererHook